spec/rubocop/cop/rspec/change_by_zero_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ChangeByZero do
+  it 'registers an offense when using `to change(object, :attribute).by(0)`' do
+    expect_offense(<<-RUBY)
+      it do
+        expect{ foo }.to change(User, :count).by(0)
+                         ^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `not_to change` over `to change.by(0)`.
+      end
+    RUBY
+
+    expect_correction(<<-RUBY)
+      it do
+        expect{ foo }.not_to change(User, :count)
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using' \
+    '`to change(object, :attribute).by(not 0)`' do
```

Comment:
This code is backtick-quoted, I'd think it means that literally `not 0`.
To be honest, I even checked if `not 0` would run and what it evaluates to 😄 
WDYT of "does not register an offense when the argument to `by` is not zero"?

Reply:
"does not register an offense when the argument to by is not zero"  looks better. I'll fix it.